content/view/view.html

```diff
@@ -3,50 +3,46 @@ <h4 class="alert-heading">Serify is going live!</h4>
   <h5 >Serify has made it to the final round of the Big Ideas @ Berkeley 2017.</h5>
   <h5 class="mb-0"><a ng-click="voteButtonClick()">Help us go live, Vote Here!</a></h5>
 </div>
-<div class="ui" style="margin:0% 10% 0% 10%;width:80% !important">
-	<div ng-show="linkname && !error" class="do-not-animate" style="text-align: center; border: 3px solid #00CED1;">
-			<h4>Serify profile:</h4>
-			<h4>{{username}}</h4>
+<div class="publicProfile">
+	<div ng-show="linkname && !error" class="do-not-animate">
+			<h4 id="publicProfileUsername">{{username}}</h4>
 	</div>
-	<div ng-show="profile" style="text-align:center">
-		<br>
+	<div id="publicProfileInfo" ng-show="linkname && !error">
 		{{profile}}
+		<hr>
 	</div>
-	<div ng-show="!linkname && !error" class="do-not-animate" style="text-align: center; border: 3px solid #00CED1;">
-			<h4>Retrieving health record verifications...</h4>
+	<div ng-show="!linkname && !error" class="do-not-animate">
+			<h4 id="publicProfileUsername">Retrieving Profile...</h4>
```

Comment:
It's unique for the public profile username text. So that when I edit the style in the CSS file I know which element it's referring to.